local-cli/runIOS/runIOS.js

```diff
@@ -63,7 +68,7 @@ function _runIOS(argv, config, resolve, reject) {
 
   const xcodebuildArgs = [
     xcodeProject.isWorkspace ? '-workspace' : '-project', xcodeProject.name,
-    '-scheme', inferredSchemeName,
+    '-scheme', (args.scheme || inferredSchemeName),
```

Comment:
Just a style suggestion, let's move the computations above the final command call, so it's easier to read and to debug when something goes wrong. In this case you can add `const scheme = args.scheme || inferredSchemeName` just after line 49.